### PR TITLE
DB-11001 Non durable ITs

### PR DESCRIPTION
**Pull request by dgomezferro**
This PR makes durability asynchronous in standalone by default (we don't flush to WAL every write, we only do that after a 1s timeout) and it removes durability for for ITs
It also removes durability for HMaster procedures (only important for HMaster crashes in the middle of a procedure execution) and makes conglomerate creation asynchronous for Create Table.

These changes reduced an IT run by 45 minutes locally (2:15 to 1:30) and branch-PR builds by 30 minutes approx.

### CODE REVIEW

**Comment by dgomezferro** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/2118/

**Comment by dgomezferro** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/2119/

**Comment by dgomezferro** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by dgomezferro** [comment]
This PR makes durability asynchronous in standalone by default (we don't flush to WAL every write, we only do that after a 1s timeout) and it removes durability for for ITs 
It also removes durability for HMaster procedures (only important for HMaster crashes in the middle of a procedure execution) and makes conglomerate creation asynchronous for Create Table.

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

**Review by martinrupp — APPROVED** [review]
cool

**Review by ipraznik-splice — APPROVED** [review]

File: platform_it/pom.xml [hunk 1]
```diff
@@ -1091,6 +1095,7 @@
             </activation>
             <properties>
                 <memberNumber>1</memberNumber>
+                <durability>ASYNC</durability>
```
Comment (hatyo) [inline platform_it/pom.xml:1098]: should this be NONE?

**Review by hatyo — APPROVED** [review]
Great change!

**Review by carolp-503 — APPROVED** [review]
+1

**Pull request merged**